autodoc/admin-api/data/admin-api.lua

```diff
@@ -437,11 +437,11 @@ return {
             {
                 "fields": {
                     "hide_credentials": {
-                        "default": false,
+                        "default": true,
```

Comment:
```suggestions
"default": false,
```

https://github.com/Kong/kong/blob/d52007a4457af151aed13585f171cb0eea5835f2/kong/plugins/key-auth/schema.lua#L18

Reply:
nice catch!